src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -279,10 +279,10 @@
     <Compile Condition="'$(TargetGroup)' != 'uap'" Include="System\Diagnostics\Process.Win32.cs">
       <SubType>Component</SubType>
     </Compile>
-    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="System\Diagnostics\Process.WinRT.cs">
-      <SubType>Component</SubType>
-    </Compile>
+    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="System\Diagnostics\Process.WinRT.cs" />
     <Compile Include="System\Diagnostics\ProcessManager.Windows.cs" />
+    <Compile Condition="'$(TargetGroup)' != 'uap'" Include="System\Diagnostics\ProcessManager.Win32.cs" />
+    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="System\Diagnostics\ProcessManager.WinRT.cs" />
```

Comment:
Nit: now that we're starting to have more of these conditions and Win32 vs WinRT files, should we create ItemGroups that have the conditions and then group the relevant files under them?